test/support/factory.ex

```diff
@@ -0,0 +1,64 @@
+defmodule ElixirBench.Factory do
+  use ExMachina.Ecto, repo: ElixirBench.Repo
+
+  alias ElixirBench.Repos
+  alias ElixirBench.Benchmarks.{Benchmark, Measurement, Job}
+
+  def repo_factory do
+    %Repos.Repo{
+      name: sequence(:name, &"ecto-#{&1}"),
+      owner: sequence(:owner, &"elixir-ecto-#{&1}")
+    }
+  end
+
+  def benchmark_factory do
+    %Benchmark{
+      name: sequence(:name, &"my_benchmark/scenario-#{&1}"),
+      repo: build(:repo)
+    }
+  end
+
+  def job_factory do
+    %Job{
+      branch_name: "master",
+      commit_sha: sequence(:commit_sha, &"ABCDEF#{&1}"),
+      commit_message: "My commit message",
+      commit_url: "git.com",
+      elixir_version: "1.5.2",
+      erlang_version: "20.1",
+      repo: build(:repo)
+    }
+  end
+
+  def measurement_factory do
+    repo = insert(:repo)
```

Comment:
iirc ex_machine correctly we could make this `build` and it should work? Might be wrong though :)